network-api/networkapi/templates/pages/buyersguide/base.html

```diff
@@ -78,22 +78,26 @@
   {% include "fragments/buyersguide/primary_nav.html" %}
   </div>
 
-  {% get_bg_home_page as home_page %}
-
-  {% block hero %}{% endblock %}
+  <div class="underneath-screen-overlay">
+    <div class="tw-relative">
+      <div class="tw-relative tw-z-10">
+        {% get_bg_home_page as home_page %}
 
-  {% block category_header %}{% endblock %}
+        {% block hero %}{% endblock %}
 
-  {% block category_nav %}
-  {% include "fragments/buyersguide/pni_mobile_nav.html" with pagetype=pagetype categories=categories current_category=current_category %}
-  {% include "fragments/buyersguide/pni_category_nav.html" with pagetype=pagetype categories=categories current_category=current_category %}
-  {% endblock category_nav %}
+        {% block category_header %}{% endblock %}
 
-  <div class="underneath-screen-overlay">
-    <main class="main-content {% block main_content_class %}pb-5{% endblock %}">
-      {% block guts %}{% endblock %}
-    </main>
+        {% block category_nav %}
+          {% include "fragments/buyersguide/pni_mobile_nav.html" with pagetype=pagetype categories=categories current_category=current_category %}
+          {% include "fragments/buyersguide/pni_category_nav.html" with pagetype=pagetype categories=categories current_category=current_category %}
+        {% endblock category_nav %}
 
+        <main class="main-content {% block main_content_class %}pb-5{% endblock %}">
+          {% block guts %}{% endblock %}
+        </main>
+      </div>
+      {% block background-parallax %}{% endblock %}
```

Comment:
What do you think about using `_` in the block name, rather than `-` to be consistent with the other block names in this template?

Reply:
done